packages/Kokkos/src/DTK_BoundingVolumeList.hpp

```diff
@@ -0,0 +1,94 @@
+/*
+  Copyright (c) 2012, Stuart R. Slattery
+  All rights reserved.
+
+  Redistribution and use in source and binary forms, with or without
+  modification, are permitted provided that the following conditions are
+  met:
+
+  *: Redistributions of source code must retain the above copyright
+  notice, this list of conditions and the following disclaimer.
+
+  *: Redistributions in binary form must reproduce the above copyright
+  notice, this list of conditions and the following disclaimer in the
+  documentation and/or other materials provided with the distribution.
+
+  *: Neither the name of the University of Wisconsin - Madison nor the
+  names of its contributors may be used to endorse or promote products
+  derived from this software without specific prior written permission.
+
+  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+  "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+  LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+  A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+  HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+  SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+  LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+  DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+  THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+  (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+  OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*/
+//---------------------------------------------------------------------------//
+/*!
+ * \brief DTK_BoundingVolumeList.hpp
+ * \brief BoundingVolume list.
+ */
+//---------------------------------------------------------------------------//
+
+#ifndef DTK_BOUNDINGVOLUMELIST_HPP
+#define DTK_BOUNDINGVOLUMELIST_HPP
+
+#include "DTK_ConfigDefs.hpp"
+
+#include <Kokkos_Core.hpp>
+
+namespace DataTransferKit
+{
+//---------------------------------------------------------------------------//
+/*!
+ * \class BoundingVolumeList.
+ *
+ * \brief Trivially-copyable bounding volume list.
+ *
+ * \tparam ViewProperties Properties of the contained Kokkos views.
+ */
+template <class... ViewProperties>
+class BoundingVolumeList
+{
+  public:
+    //@{
+    //! Type aliases.
+    using ViewTraits = typename Kokkos::ViewTraits<int, ViewProperties...>;
+    using ExecutionSpace = typename ViewTraits::execution_space;
+    using MemorySpace = typename ViewTraits::memory_space;
+    using Device = typename ViewTraits::device_type;
+    //@}
+
+    //! The coordinates of the bounding volumes that are locally-owned by this
+    //! MPI rank. This view is rank-3 and should be sized as (number of
+    //! bounding volumes, spatial dimension, 2 [for low and high coordinate]).
+    Kokkos::View<Coordinate ***, ViewProperties...> bounding_volumes;
```

Comment:
changing to be `**[2]`. We can coordinate with the search when we are going to merge #183